boards/esp32s2-mini/Makefile.dep

```diff
@@ -0,0 +1,2 @@
+include $(RIOTBOARD)/common/makefiles/stdio_tinyusb_cdc_acm.dep.mk
```

Comment:
Shouldn't it also work with `stdio_cdc_acm`? The problem is that the board can't be used with `periph_usbdev` which is supported by ESP32-S2 if `stdio_tinyusb_cdc_acm` is used. Maybe we should use a different approach that selects `stdio_tinyusb_cdc_acm` only if `tinyusb_device` is already enabled by another `tinyusb_*` module and `stdio_cdc_acm` otherwise.

Here the discussion with @chrysn about the relationship between tinyUSB and USBUS pops up again.

Reply:
But we have no `periph_usbdev` implementation for ESP32

Reply:
Yes, but we have it for ESP32-S2 and ESP32-S3. ESP32 doesn't have an USB controller at all.

Reply:
Hm turns out that does not work

```
[67809.301628] usb 1-1.3: new full-speed USB device number 44 using ehci-pci
[67809.410682] usb 1-1.3: device descriptor read/all, error -32
[67809.417658] usb 1-1-port3: attempt power cycle
[67810.025642] usb 1-1.3: new full-speed USB device number 45 using ehci-pci
[67810.058690] usb 1-1.3: device descriptor read/8, error -32
[67810.186688] usb 1-1.3: device descriptor read/8, error -32
[67810.377642] usb 1-1.3: new full-speed USB device number 46 using ehci-pci
[67810.410669] usb 1-1.3: device descriptor read/8, error -32
[67810.538654] usb 1-1.3: device descriptor read/8, error -32
[67810.649812] usb 1-1-port3: unable to enumerate USB device
```